recommended_ruleset.js

```diff
@@ -205,8 +205,8 @@ module.exports = {
         "no-missing-visibility-modifiers": false, // use tslint member-access rule instead
         "no-multiple-var-decl": false,         // use tslint one-variable-per-declaration rule instead
         "no-switch-case-fall-through": false,  // now supported by TypeScript compiler
-        "no-unused-imports": false,            // use tslint no-unused-variable rule instead
-        "no-unused-variable": false,           // now supported by TypeScript 2.0 compiler
-    }
+        "no-unused-imports": false             // Use the tsc compiler options --noUnusedParameters and --noUnusedLocals instead
+//        "no-unused-variable": true             // Use the tsc compiler options --noUnusedParameters and --noUnusedLocals instead
```

Comment:
So I just spotted line 209 above. It was missing from the npm package but even after I moved it and disabled the rule I still get warnings.